_docs-sources/intro/overview/business-case.md

```diff
@@ -0,0 +1,56 @@
+# Business case
+
+## Problem
+
+Every company needs infrastructure to run its software, but deploying and maintaining infrastructure can be complicated, error-prone and time-consuming. Also, most companies have the same core set of infrastructure needs, so redundantly implementing what is already built is a poor use of company resources.
+
+## Traditional solutions
+
+In the past, companies dealing with the complexity of infrastructure had two primary options:
+
+1. **Outsource DevOps.** Outsourcing options hide the lower-level details so it's easy to get started, but hard to customize, debug and scale.
+1. **Build everything from scratch.** This gives the team full control, but at the significant cost of assembling everything themselves.
+
+Gruntwork offers a third option: **Accelerated DevOps.**
+
+## The Gruntwork solution
+
+In the Gruntwork model, you retain full control of your infrastructure and use Gruntwork to _accelerate_ your DevOps capabilities by using pre-built, best-in-class implementations that can be adapted to a wide range of common use cases, and which come with extensive documentation, automated tests, and commercial support.
+
+### Benefits
+
+- **Speed.** Launch faster by not re-inventing the wheel.
+- **Confidence.** Setup infrastructure the "right" way upfront.
+- **Reduce cost.** Offload significant ongoing maintenance costs and gain access to future updates by leveraging the Gruntwork engineering team.
+- **Community.** Get access to an active community all using the same building blocks.
+- **Resilience.** Streamline the way your future team members ramp up.
```

Comment:
"Resilience" doesn't feel like quite the right term here. This sounds more along the lines of "battle-tested, well-maintained code."